pkg/bundle/oidc.go

```diff
@@ -49,8 +49,8 @@ func configureOidc(repo string, client api.Client, recipe *api.Recipe, ctx map[s
 		},
 	}
 	mergeOidcAttributes(inst, oidcSettings)
-
-	return client.OIDCProvider(inst.Id, oidcSettings)
+	err = client.OIDCProvider(inst.Id, oidcSettings)
+	return api.GetErrorResponse(err, "OIDCProvider")
```

Comment:
Shouldn't we check for `if err != nil` here. Similar to in [pkg/crypto/age.go](https://github.com/pluralsh/plural-cli/pull/277/files#diff-42e3a757ec072a799849e121c4cd26da264e442216ec0369c1f7c477fdd983f9)